controllers/flinkcluster/flinkcluster_converter.go

```diff
@@ -292,6 +292,7 @@ func newJobManagerService(flinkCluster *v1beta1.FlinkCluster) *corev1.Service {
 	switch jobManagerSpec.AccessScope {
 	case v1beta1.AccessScopeCluster:
 		jobManagerService.Spec.Type = corev1.ServiceTypeClusterIP
+		jobManagerService.Annotations = jobManagerSpec.ServiceAnnotations
```

Comment:
We could set the annotations earlier right in L#276

Reply:
Done